docs/wiki/Known issues.md

```diff
@@ -53,15 +54,15 @@ Further details are also provided in issue [#1342](https://github.com/Azure/Reso
 
 ## recovery-services/vault
 
-The Recovery Services Vaults module does not currently attach the content of the identity property correctly when both user- and systemassigned identity fields are selected.
+The Recovery Services Vaults module does not currently attach the content of the identity property correctly when both user and system-assigned identity fields are selected.
```

Comment:
```suggestion
The Recovery Services Vaults module does not currently attach the content of the identity property correctly when both user and system-assigned identity fields are selected.
```
```suggestion
The Recovery Services Vaults module does not currently attach the content of the identity property correctly when both user- and system-assigned identity fields are selected.
```
If you add the `-` for system-assigned, I guess we can keep on using it for `user` too :D